### ISSUE TITLE
Mission checksum for UI synchronisation

### ISSUE THREAD

**Issue by holmnikolaj** [PX4/Firmware#12399]
To make sure that the mission on the drone is identical to the one shown in the UI You have to download all the mission to check. This can take quite a while for a long mission just to check. 

If the first item in the mission were a checksum the synchronization could be check by just downloading one mission item.

**Comment by dagar** [comment]
Yes absolutely! I've wanted to do this for a long time, but the blocker is getting the mavlink spec updated.

**Comment by dagar** [comment]
Have you thought about how we could do this, including where to extend mavlink?
